nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -71,6 +71,26 @@ def add_statistic_point(self, statistic_point: StatisticPoint) -> None:
 
             self.data[target_node_name].append(statistic_point)
 
+    def remove_statistic_points(self, algorithm_name: str) -> None:
```

Comment:
So, what about the integration of this method into the pipeline? As described in the https://github.com/openvinotoolkit/nncf/issues/2557.

Reply:
I did, though
Check `pipeline.py` 
(I'd send a screenshot but I am using phone right now, sorry)

Reply:
Yes, I agree. It's just that it _is_ in this PR
If you can confirm that it's not, I can try opening another PR

Reply:
OMG I found the problem. That line got removed in one of the commits😭
I shall fix it soon, so sorry!

Reply:
OMG I found the problem. That line got removed in one of the commits😭
I shall fix it soon, so sorry!